### PR TITLE
Added donut chart component

### PR DESCRIPTION
**Pull request by justin0022**

### CODE REVIEW

File: src/components/d3/createDonutChart.js [hunk 1]
```diff
@@ -3,5 +3,32 @@ import { adjustViewport } from '../../util/chart'
 import { margin } from '../../constants/chartConstants'
 
 function createDonutChart ({ data, width, height, el, tip }) {
+  const [aWidth, aHeight] = adjustViewport(width, height, margin)
 
-}
\ No newline at end of file
+  const radius = Math.min(aWidth, aHeight) / 2
+  const arc = d3.arc().innerRadius(radius * 0.67).outerRadius(radius - 1)
```
Comment (soohyunkim) [inline src/components/d3/createDonutChart.js]: Why 0.67? Is this a constant?

Comment (justin0022) [reply src/components/d3/createDonutChart.js]: Good point. Those constants (the 0.67 and the 1) control the thickness of the donut. Should I assign them to variables? Not sure what I would call them.

Comment (soohyunkim) [reply src/components/d3/createDonutChart.js]: Maybe `thicknessMultiplier` for `0.67`? Not sure either haha.

Comment (justin0022) [reply src/components/d3/createDonutChart.js]: Ya that could work. I think the `radius - 1` also controls the thickness too. One basically controls the radius of the inner circle, and the other controls the outer circle.

**Pull request merged**